recipe/meta.yaml

```diff
@@ -26,17 +24,18 @@ package:
 
 source:
   url: https://github.com/grpc/grpc/archive/v{{ version }}.tar.gz
-  sha256: b16992aa1c949c10d5d5ce2a62f9d99fa7de77da2943e643fb66dcaf075826d6
+  sha256: 9647220c699cea4dafa92ec0917c25c7812be51a18143af047e20f3fb05adddc
   patches:
-    - aarch64-tcp-header.patch  # [aarch64]
     - force-protoc-executable.patch
   # You can use module version of re2 if you add this and chage build.sh:
   # - git_url: https://github.com/google/re2
   #   git_ref: aecba11114cf1fac5497aeb844b6966106de3eb6
   #   folder: third_party/re2
 
 build:
-  number: 5
+  number: 0
+  # Anaconda's s390x does not contain depdendencies for abseil-cpp or re2, this package is skipped for s390x at this time.
+  skip: True  # [win32 or s390x]
```

Comment:
I see we have previously supported win32, why are we skipping it now?

Reply:
Latest protobuf recipe does not support 32bit.